korio/src/jsMain/kotlin/com/soywiz/korio/runtime/node/JsRuntimeNode.kt

```diff
@@ -163,6 +165,7 @@ private class NodeJsAsyncClient(val coroutineContext: CoroutineContext) : AsyncC
 private class NodeJsAsyncServer : AsyncServer {
     private val net = require_node("net")
     private var server: dynamic = null
+    private val logger = Logger("NodeJsAsyncServer")
```

Comment:
Maybe inside a companion object, so the calling happens just once?

Reply:
done (also for asyncClient)